crates/red_knot_module_resolver/src/resolver.rs

```diff
@@ -42,7 +41,7 @@ pub(crate) fn resolve_module_query<'db>(
     db: &'db dyn Db,
     module_name: internal::ModuleNameIngredient<'db>,
 ) -> Option<Module> {
-    let _ = tracing::trace_span!("resolve_module", module_name = ?module_name.debug(db)).enter();
+    let _span = tracing::trace_span!("resolve_module", ?module_name).entered();
```

Comment:
I don't mind it too much. One thing I think would be worth exploring is if it's possible to add a `tracing` feature to `salsa-macros` or change the macro so that it applies other attributes correctly